packages/notification-center-vue/src/lib/vue-component-lib/utils.ts

```diff
@@ -0,0 +1,204 @@
+import { VNode, defineComponent, getCurrentInstance, h, inject, ref, Ref } from 'vue';
```

Comment:
Is this file made from scratch by you or autogenerated by Vue? Lazy me wants to know to review it or not. 😛